### PR TITLE
Make it possible to customize the ACM cert validation wait time.

### PR DESCRIPTION
**Pull request by xiaket**
We are using AWS lambda to create/delete the acm certificates and doing the validation via DNS, when writing unittest for these lambdas we'll have to wait 60 seconds. This PR make it possible to customize this wait time.

### CODE REVIEW

File: moto/acm/models.py [hunk 1]
```diff
@@ -324,13 +325,15 @@ def validate_chain(self):
             )
 
     def check(self):
-        # Basically, if the certificate is pending, and then checked again after 1 min
-        # It will appear as if its been validated
+        # Basically, if the certificate is pending, and then checked again after a
+        # while, it will appear as if its been validated. The default wait time is 60
+        # seconds but you can set an environment to change it.
+        wait = os.environ.get("MOTO_ACM_VALIDATION_WAIT", 60)
```
Comment (bblommers) [inline moto/acm/models.py:331]: This should be moved into `moto/settings.py`, where all our env variables are stored

**Review by bblommers — CHANGES_REQUESTED** [review]
Hey @xiaket, thanks for the PR!
Would you mind adding a unit test for this? See this test for an example on how we test env vars in other services: https://github.com/spulec/moto/blob/d3ad9d66866413670c31f36d54e939647feedeb1/tests/test_stepfunctions/test_stepfunctions.py#L933

File: tests/test_acm/test_acm.py [hunk 2]
```diff
@@ -530,6 +545,36 @@ def test_request_certificate_issued_status():
         resp["Certificate"]["Status"].should.equal("ISSUED")
 
 
+@mock_acm
+@mock.patch.dict(os.environ, {"MOTO_ACM_VALIDATION_WAIT": 3})
+def test_request_certificate_issued_status_with_wait_in_envvar():
+    # After requesting a certificate, it should then auto-validate after 1 minute
```
Comment (bblommers) [inline tests/test_acm/test_acm.py:551]: Still need to use the SkipTest here, otherwise our ServerMode tests wil fail

**Comment by bblommers** [comment]
Make sure that  you use `black==19.10b0` for linting, as later versions have a few bugs

**Comment by xiaket** [comment]
> Make sure that you use `black==19.10b0` for linting, as later versions have a few bugs

Cheers mate, I'm having trouble with this. :)

**Comment by xiaket** [comment]
Thanks @bblommers. I ended up not mocking the envvar directly but mocked the value from settings(because there's a cache somewhere that I cannot figure out). Hope it looks good now. Please let me know if you have any follow up suggestions/comments. I appreciate your help!

**Review by bblommers — APPROVED** [review]
LGTM @xiaket - thanks again!

**Pull request merged**